pkg/api/postage.go

```diff
@@ -164,6 +165,7 @@ type bucketData struct {
 func (s *Service) postageGetStampsHandler(w http.ResponseWriter, r *http.Request) {
 	isAll := strings.ToLower(r.URL.Query().Get("all")) == "true"
 	resp := postageStampsResponse{}
+	var isExpired bool
```

Comment:
We should move this variable inside the for loop. The scope of this variable is wrong.